services/rollout-service/pkg/argo/argo.go

```diff
@@ -195,6 +198,16 @@ func (a ArgoAppProcessor) CreateOrUpdateApp(ctx context.Context, overview *api.G
 	}
 }
 
+func (a *ArgoAppProcessor) IsSelfManagedFilterActive(team string) (bool, error) {
+	if len(a.ManageArgoAppsFilter) > 1 && slices.Contains(a.ManageArgoAppsFilter, "*") {
```

Comment:
This code is not tested by any unit test. Try replacing this method body with `panic("no covered")` and all tests still pass. Please add tests.

Reply:
I removed the duplicate function in the test file, re-added the interface (so that we can test the function from the go file), added the interface as an argument to the function so that we can test both cases.